wfe/wfe.go

```diff
@@ -836,6 +836,12 @@ func (wfe *WebFrontEndImpl) updateChallenge(
 				fmt.Sprintf("Failed to validate challenge: %q", err)), response)
 		return
 	}
+	response.Header().Add("Link", link(existingChal.Authz.URL, "up"))
```

Comment:
Let's leave it out for now. I'd like to discuss with the ACME list whether all these link relations are really necessary. This one is for now, because the Python `acme` module expects it (thought it could be modified to not).